### ISSUE TITLE
Simpler Named Exports

### ISSUE THREAD

**Issue by skellock** [infinitered/gluegun#312]
In JavaScript it was kinda easier to [wrap up exports in an object](https://github.com/infinitered/gluegun/blob/0e368e24fdc0a80d0098dc1a6bf5750db8893d88/src/toolbox/string-tools.ts#L70-L99), however in TypeScript, I find it often simpler to just do it inline.  It will compile down to the right thing.

For example:

```
function pascalCase(value) {
  return pipe(camelCase, upperFirst)(value)
}
```

Would change to:

```
export function pascalCase(value) {
  return pipe(camelCase, upperFirst)(value)
}
```

And then you could drop the [whole bottom half](https://github.com/infinitered/gluegun/blob/0e368e24fdc0a80d0098dc1a6bf5750db8893d88/src/toolbox/string-tools.ts#L70-L99) of that file.

**Comment by jamonholmgren** [comment]
I couldn't figure out how to export [this part](https://github.com/infinitered/gluegun/blob/0e368e24fdc0a80d0098dc1a6bf5750db8893d88/src/toolbox/string-tools.ts#L19-L28) of the file. Any suggestions?

**Comment by jamonholmgren** [comment]
Fixed in #319 and #324 .
